bundles/org.openhab.binding.mybmw/src/main/resources/OH-INF/thing/charge-statistics-channel-types.xml

```diff
@@ -11,11 +11,19 @@
 		<item-type>Number:Energy</item-type>
 		<label>Energy Charged</label>
 		<description>Total energy charged in current month</description>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Energy</tag>
+		</tags>
 		<state pattern="%d %unit%" readOnly="true"/>
 	</channel-type>
 	<channel-type id="statistic-sessions-channel">
 		<item-type>Number</item-type>
 		<label>Charge Sessions</label>
 		<description>Number of charging sessions this month</description>
+		<tags>
+			<tag>Control</tag>
```

Comment:
This is not a control channel, but statistics. What is most appropropriate: Status or Measurement?

Reply:
Just for info: On Number channels the auto tagger sets `Control` tag if the channel is not read only, and `Measurement` if it is. Similar on Switch channels `Status` vs `Switch`. But on String channels it shows `Status` unless the channel id, label, description or category texts contain action words like command, change, set, etc.

Reply:
Probably `Status`..